packages/core/src/extensions/Comments/CommentMark.ts

```diff
@@ -0,0 +1,45 @@
+import { Mark, mergeAttributes } from "@tiptap/core";
+
+export const CommentMark = Mark.create({
+  name: "comment",
+  excludes: "",
+  inclusive: false,
+  keepOnSplit: true,
+  group: "blocknoteIgnore", // ignore in blocknote json
+
+  addAttributes() {
+    // Return an object with attribute configuration
+    return {
+      // TODO: check if needed
+      orphan: {
+        parseHTML: (element) => !!element.getAttribute("data-orphan"),
+        renderHTML: (attributes) => {
+          return (attributes as { orphan: boolean }).orphan
+            ? {
+                "data-orphan": "true",
+              }
+            : {};
+        },
+        default: false,
+      },
+      threadId: {
+        parseHTML: (element) => element.getAttribute("data-bn-thread-id"),
+        renderHTML: (attributes) => {
+          return {
+            "data-bn-thread-id": (attributes as { threadId: string }).threadId,
+          };
+        },
+        default: "",
+      },
+    };
+  },
+
+  renderHTML({ HTMLAttributes }: { HTMLAttributes: Record<string, any> }) {
```

Comment:
Depending on if you want to support copy-paste of comments, maybe parseHTML?

Reply:
Good point!

I think copy paste will need to be tested extensively :) for example, you could paste a comment that's not in the store, etc